process/io/plot_radial_build.py

```diff
@@ -264,7 +264,6 @@ def main(args=None):
         "tbrmin",
         "bt",
         "coreradius",
-        "fimpvar",
```

Comment:
How come below you have done `"Obsolete",  # Removed` but not here? Could this affect indexing?